packages/tools/addon/helpers/cs-field-editor-for.js

```diff
@@ -1,9 +1,18 @@
-import Ember from 'ember';
-
+import { helper } from '@ember/component/helper';
 import { fieldType } from '@cardstack/rendering/helpers/cs-field-type';
+// @TODO: expose public api
+import metaForField from '@cardstack/rendering/-private/meta-for-field';
```

Comment:
Maybe we should move this to `@cardstack/models` and expose it publicly?

Reply:
Agreed. It doesn't need to be private.